lib/floe/workflow/states/pass.rb

```diff
@@ -28,6 +28,7 @@ def finish
           input              = process_input(context.input)
           context.output     = process_output(input, result)
           context.next_state = end? ? nil : @next
+          context.next_state = end? ? nil : @next
```

Comment:
oops

Reply:
Thanks, was just trying to get something working last night :sweat_smile: